### PR TITLE
Update version @babel/runtime

### PR DESCRIPTION
**Pull request by micro-chipset**
Please update the @babel/runtime version by removing the hard-coded version.
Many packages use a newer version because of this me have to use multiple versions of the  @babel/runtime package

## Bug

- [n] Related issues linked using `fixes #number`
- [n] Integration tests added
- [n] Errors have helpful link attached, see `contributing.md`

## Feature

- [n] Implements an existing feature request or RFC. Make sure the feature request has been accepted for implementation before opening a PR.
- [n] Related issues linked using `fixes #number`
- [n] Integration tests added
- [n] Documentation added
- [n] Telemetry added. In case of a feature if it's used or not.
- [n] Errors have helpful link attached, see `contributing.md`

## Documentation / Examples

- [y] Make sure the linting passes

### CODE REVIEW

**Review by abood640 — DISMISSED** [review]

**Comment by micro-chipset** [comment]
@huozhi  @ijjk Please review.

File: packages/next/package.json [hunk 1]
```diff
@@ -64,7 +64,7 @@
     ]
   },
   "dependencies": {
-    "@babel/runtime": "7.15.3",
+    "@babel/runtime": "^7.15.3",
```
Comment (timneutkens) [inline packages/next/package.json:67]: Version locking is intentional to ensure that the version that was tested is being used. Feel free to update the dependency itself.

**Review by timneutkens — DISMISSED** [review]
Left a comment

File: packages/next/package.json (hunk 1 above)
Comment (micro-chipset) [reply packages/next/package.json:67]: @timneutkens I would like the assembly to have one version of the @babel/runtime package. In many packages, the version is not fixed and this leads to an increase in the build size.

Comment (timneutkens) [reply packages/next/package.json:67]: You can use yarn resolutions for that, but in practice dependencies should not have @babel/runtime anyway 🤔

Comment (micro-chipset) [reply packages/next/package.json]: @timneutkens update fix version

**Review by ijjk — CHANGES_REQUESTED** [review]
Hi, this also needs to be updated in the `yarn.lock` which should only require running `yarn` in the repo root.

**Comment by micro-chipset** [comment]
> Hi, this also needs to be updated in the `yarn.lock` which should only require running `yarn` in the repo root.

@ijjk Update yarn.lock

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Build</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | micro-chipset/next.js canary | Change |
| - | - | - | - |
| buildDuration | 16.9s | 16.9s | -49ms |
| buildDurationCached | 4.1s | 4s | -57ms |
| nodeModulesSize | 183 MB | 183 MB | ⚠️ +2.82 kB |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | micro-chipset/next.js canary | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 3.812 | 3.739 | -0.07 |
| / avg req/sec | 655.78 | 668.7 | +12.92 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 2.122 | 2.111 | -0.01 |
| /error-in-render avg req/sec | 1178.23 | 1184.33 | +6.1 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | micro-chipset/next.js canary | Change |
| - | - | - | - |
| 779.HASH.js gzip | 179 B | 179 B | ✓ |
| framework-HASH.js gzip | 42.2 kB | 42.2 kB | ✓ |
| main-HASH.js gzip | 26.9 kB | 26.9 kB | ✓ |
| webpack-HASH.js gzip | 1.45 kB | 1.45 kB | ✓ |
| Overall change | 70.8 kB | 70.8 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | micro-chipset/next.js canary | Change |
| - | - | - | - |
| polyfills-a4..dd70.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | micro-chipset/next.js canary | Change |
| - | - | - | - |
| _app-HASH.js gzip | 977 B | 977 B | ✓ |
| _error-HASH.js gzip | 194 B | 194 B | ✓ |
| amp-HASH.js gzip | 311 B | 311 B | ✓ |
| css-HASH.js gzip | 328 B | 328 B | ✓ |
| dynamic-HASH.js gzip | 2.67 kB | 2.67 kB | ✓ |
| head-HASH.js gzip | 351 B | 351 B | ✓ |
| hooks-HASH.js gzip | 918 B | 918 B | ✓ |
| image-HASH.js gzip | 4.15 kB | 4.15 kB | ✓ |
| index-HASH.js gzip | 260 B | 260 B | ✓ |
| link-HASH.js gzip | 1.66 kB | 1.66 kB | ✓ |
| routerDirect..HASH.js gzip | 320 B | 320 B | ✓ |
| script-HASH.js gzip | 386 B | 386 B | ✓ |
| withRouter-HASH.js gzip | 319 B | 319 B | ✓ |
| bb14e60e810b..30f.css gzip | 125 B | 125 B | ✓ |
| Overall change | 13 kB | 13 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | micro-chipset/next.js canary | Change |
| - | - | - | - |
| _buildManifest.js gzip | 494 B | 494 B | ✓ |
| Overall change | 494 B | 494 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | micro-chipset/next.js canary | Change |
| - | - | - | - |
| index.html gzip | 539 B | 539 B | ✓ |
| link.html gzip | 551 B | 551 B | ✓ |
| withRouter.html gzip | 533 B | 533 B | ✓ |
| Overall change | 1.62 kB | 1.62 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Default Build with SWC</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | micro-chipset/next.js canary | Change |
| - | - | - | - |
| buildDuration | 8.8s | 8.6s | -132ms |
| buildDurationCached | 4.1s | 4.1s | -70ms |
| nodeModulesSize | 183 MB | 183 MB | ⚠️ +2.82 kB |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | micro-chipset/next.js canary | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 3.764 | 3.77 | ⚠️ +0.01 |
| / avg req/sec | 664.25 | 663.16 | ⚠️ -1.09 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 2.09 | 2.077 | -0.01 |
| /error-in-render avg req/sec | 1196.11 | 1203.94 | +7.83 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | micro-chipset/next.js canary | Change |
| - | - | - | - |
| 675-HASH.js gzip | 13.8 kB | 13.8 kB | ✓ |
| 770.HASH.js gzip | 178 B | 178 B | ✓ |
| framework-HASH.js gzip | 50.7 kB | 50.7 kB | ✓ |
| main-HASH.js gzip | 34.9 kB | 34.9 kB | ✓ |
| webpack-HASH.js gzip | 1.64 kB | 1.64 kB | ✓ |
| Overall change | 101 kB | 101 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | micro-chipset/next.js canary | Change |
| - | - | - | - |
| polyfills-a4..dd70.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | micro-chipset/next.js canary | Change |
| - | - | - | - |
| _app-HASH.js gzip | 1.33 kB | 1.33 kB | ✓ |
| _error-HASH.js gzip | 180 B | 180 B | ✓ |
| amp-HASH.js gzip | 315 B | 315 B | ✓ |
| css-HASH.js gzip | 331 B | 331 B | ✓ |
| dynamic-HASH.js gzip | 2.79 kB | 2.79 kB | ✓ |
| head-HASH.js gzip | 356 B | 356 B | ✓ |
| hooks-HASH.js gzip | 637 B | 637 B | ✓ |
| image-HASH.js gzip | 573 B | 573 B | ✓ |
| index-HASH.js gzip | 262 B | 262 B | ✓ |
| link-HASH.js gzip | 2.2 kB | 2.2 kB | ✓ |
| routerDirect..HASH.js gzip | 326 B | 326 B | ✓ |
| script-HASH.js gzip | 393 B | 393 B | ✓ |
| withRouter-HASH.js gzip | 322 B | 322 B | ✓ |
| bb14e60e810b..30f.css gzip | 125 B | 125 B | ✓ |
| Overall change | 10.1 kB | 10.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | micro-chipset/next.js canary | Change |
| - | - | - | - |
| _buildManifest.js gzip | 511 B | 511 B | ✓ |
| Overall change | 511 B | 511 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | micro-chipset/next.js canary | Change |
| - | - | - | - |
| index.html gzip | 539 B | 539 B | ✓ |
| link.html gzip | 551 B | 551 B | ✓ |
| withRouter.html gzip | 532 B | 532 B | ✓ |
| Overall change | 1.62 kB | 1.62 kB | ✓ |

</details>

</details>
Commit: d420ee021d8cf9c00ae455fdfc9ee3ac3211412a

**Review by ijjk — APPROVED** [review]

**Pull request merged**